File: modul-4/Soal-2/PRAK402-2310817110013-Randy_Febrian.py
angka = int(input())

for i in range(1, angka + 1):
    if i % 2 != 0:
        print(i, end=' ')
print()

for j in range(angka, 0, -1):
    if j % 2 == 0:
        print(j, end=' ')

File: modul-4/Soal-2/PRAK402-2310817110013-Randy_Febrian.c
#include <stdio.h>

int main(){
    int angka;
    scanf("%d", &angka);

    for(int i = 1; i <= angka; i++){
        if(i % 2 != 0){
            printf("%d ", i);
        }
    }
    printf("\n");

    for(int j = angka; j >= 1; j--){
        if(j % 2 == 0){
            printf("%d ", j);
        }
    }
    return 0;
}